packages/editor-ui/src/mixins/nodeBase.ts

```diff
@@ -304,9 +308,13 @@ export const nodeBase = mixins(
 									position: newNodePosition,
 								},
 							};
-
+							const oldPosition = node.position;
+							if (oldPosition[0] !== newNodePosition[0] || oldPosition[1] !== newNodePosition[1]) {
+								this.historyStore.pushCommandToUndo(new MoveNodeCommand(node.name, oldPosition, newNodePosition, this));
+							}
 							this.workflowsStore.updateNodeProperties(updateInformation);
```

Comment:
can you also wrap this with a position check? only update node properties if position has changed.. I think this will fix the bug where if you open a node sometimes, it will say workflow has unsaved changes

Reply:
Sure.